Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStoreCollection.mm

```diff
@@ -104,15 +104,6 @@
     }
 }
 
-std::unique_ptr<RemoteLayerBackingStore> RemoteLayerBackingStoreCollection::createRemoteLayerBackingStore(PlatformCALayerRemote* layer)
-{
-    // We currently only create a single type of backing store based on the global setting, but
-    // it should be fine to mix both types in the same collection.
```

Comment:
Based on the new code in `PlatformCALayerRemote::ensureBackingStore()`, was this comment wrong?

Reply:
Do we store the `RemoteLayerBackingStore` in a collection? Can't we assert somewhere all the backing stores in this collection have the same `ProcessModel`

Reply:
> was this comment wrong

I forgot to remove this comment when I added the "needsPlatformContext" check.

> Can't we assert somewhere all the backing stores in this collection have the same ProcessModel

No! Because they don't. Matt made it possible to have mixed collections, and we're making use of that fact for PDF -- that's where this bug came from!